homeassistant/components/system_log/__init__.py

```diff
@@ -172,17 +188,14 @@ def emit(self, record):
         default upper limit is set to 50 (older entries are discarded) but can
         be changed if needed.
         """
-        if record.levelno >= logging.WARN:
```

Comment:
Is this limit now done elsewhere, by setting the limit of the queue handler?

Reply:
Thanks for looking

It's set on line 209:

queue_handler.setLevel(logging.WARN)